packages/vendure-plugin-accept-blue/src/api/accept-blue-service.ts

```diff
@@ -62,6 +63,7 @@ export class AcceptBlueService implements OnApplicationBootstrap {
     private readonly customerService: CustomerService,
     private readonly entityHydrator: EntityHydrator,
     private connection: TransactionalConnection,
+    private historyService: HistoryService,
```

Comment:
Can you remove this, and the import, as this change doesnt require historyService anymore?